fastlane_core/lib/fastlane_core/ui/github_issue_inspector_reporter.rb

```diff
@@ -0,0 +1,49 @@
+module Fastlane
+  # Adds all the necessary emojis (obv)
+  #
+  class InspectorReporter
+    NUMBER_OF_ISSUES_INLINE = 3
+
+    # Called just as the investigation has begun.
+    def inspector_started_query(query, inspector)
+      puts ""
+      puts "Looking for related GitHub issues on #{inspector.repo_owner}/#{inspector.repo_name}..."
+      puts "Search query: #{query}" if $verbose
+      puts ""
+    end
```

Comment:
Nah, it's all synchronous - it's really tough to do async in ruby. The delegate pattern is because CP uses it's own class for UI, and so using `puts` isn't possible, needed to pass to something else to do the output